exampleSite/content/_index/items-2/customizble.md

```diff
@@ -0,0 +1,7 @@
++++
+title = "Customizable"
+weight = 10
+icon = "fas fa-pencil-alt"
++++
+
+We use Bootstrap which allows for cutomizing every aspect of the style using variables
```

Comment:
Is that feature really exposed for a normal user? Might actually skip this feature.

Reply:
If resource pipeline is fixed for netlify and other hosts then the feature would be there. #268

Reply:
True, but currently it's not part of the feature set ;)